libraries/testing/tester.cpp

```diff
@@ -285,7 +283,9 @@ namespace eosio::testing {
                preactivate_all_builtin_protocol_features();
             }
             produce_block();
-            set_bios_contract();
+            if (policy == setup_policy::full || policy == setup_policy::full_except_do_not_transition_to_savanna ) {
```

Comment:
Maybe add a comment here why bios contract might be needed for `setup_policy::full_except_do_not_transition_to_savanna` (to transition to Savanna later).